projects/ui-particles-angular/src/lib/gio-form-json-schema/gio-form-json-schema.stories.ts

```diff
@@ -47,6 +47,8 @@ import { mqttAdvancedExample } from './json-schema-example/mqtt-advanced';
 import { httpProxyExample } from './json-schema-example/http-proxy';
 import { webhookAdvancedExample } from './json-schema-example/webhook-advanced';
 import { codeEditorExample } from './json-schema-example/code-editor';
+import { displayIfExample } from './json-schema-example/displayIf';
```

Comment:
if we have `disableIf`, should it be better to have a `hideIf` ? 🤔

Reply:
it's a question of usage
I asked myself the same haha, 

but for the moment we need to display if we're in the right condition. because if we're not in the right condition we don't want to see the field. if no condition you see it

And the disabled is used in the same way, but reversed. By default, it's enabled, and in a specific case, when it's not consistent, it's deactivated. 

There are not the same needs IMHO. I find them more "logical" in this sense, but we can add them in others if necessary. I also didn't introduce the possibility of "negation" for now